CONTRIBUTING.md

```diff
@@ -129,16 +129,16 @@ You should now be able to run `make proto-gen` from inside the root Tendermint d
 
 ### Visual Studio Code
 
-If you are a VS Code user, you may want to add the following to your `.vscode/settings.json`:  
+If you are a VS Code user, you may want to add the following to your `.vscode/settings.json`:
```

Comment:
_Workspace_ settings. I don't think we should have this in a global settings.

Reply:
Are you referring to my Vim stripping whitespaces or vscode settings?

Reply:
Sorry, I misread this!